py4DSTEM/process/calibration/ellipse.py

```diff
@@ -602,4 +604,99 @@ def compare_double_sided_gaussian(data, p, power=1, mask=None):
 
     return
 
+## Create look up table for background subtraction
+def get_1Dbackground(data, qx0, qy0 ,e, phi, 
+                    mask_update_iter=3, 
+                    min_relative_threshold = 4, 
+                    smoothing_window_size = None, 
```

Comment:
add a boolean flag `smoothing=False`, and add a default odd value for the window size